feeds/wlan-ap/luci/luci-theme-tip/htdocs/luci-static/tip/cascade.css

```diff
@@ -1,4 +1,12 @@
-@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+/* latin */
```

Comment:
Only added font's for "latin" since I assume we don't care about "cyrillic-ext", "cyrillic", "vietnamese" and "latin-ext" font sets. We could also change out the font for a standard web font to avoid this issue all together.

Reply:
OK